### ISSUE TITLE
Readme should describe DoF community

### ISSUE THREAD

**Issue by sdmg15** [DevOfFuture/dof-conf#2]

**Comment by valerymelou** [comment]
Don't you think the README should be about how running the project locally?
I'm just asking.

**Comment by sdmg15** [comment]
Not really, we can create another file BUILDING.md or something like that,
Or we can divide the readme into sections.

About running I don't know much about it :sweat_smile:  so if you can provide some info about it it'll be great.

**Comment by valerymelou** [comment]
Yep was planning to do that too. I think I'll assign myself the issue too :wink:
